develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -0,0 +1,381 @@
+  severityNumber: 11,
+});
+```
+
+Beyond the two specified sets of methods, SDKs are free to add any extra helpers as they feel is necessary. For example, they could choose to add specialized decorators or helpers for string template creation.
+
+Below are some example SDK implementations to get you started. These are not finalized versions of the API and individual SDK authors should ensure the logging APIs best fit their platforms. When an SDK implements the logging API, this section should be updated with the SDK's specific implementation.
+
+#### JavaScript
+
+```jsx
+// Browser JavaScript - need to rely on tagged template literals for string templating
+Sentry.logger.info(Sentry.logger.fmt`Adding item ${itemId} for user ${userId}`);
+
+// Server-side (Node.js, Bun, Deno) with printf-like syntax
+Sentry.logger.info("Adding item %s for user %s", [itemId, userId], {
+  extra: 123,
+});
+
+Sentry.logger.emit({
+  level: "info",
+  message: "Adding item for user",
+  attributes: {
+    itemId,
+    userId,
+    extra: 123,
+  },
+});
+```
+
+#### Python
+
+```python
+# With f-string like syntax
+Sentry.logger.info('Adding item {item} for user {user}', item=item_id, user=user_id, attributes={ 'extra': 123 });
+
+Sentry.logger.emit(
+  level='info',
+  message='Adding item for user',
+  attributes={
+    'item_id': item_id,
+    'user_id': user_id,
+    extra: 123
+  },
+)
+```
+
+#### PHP
+
+```php
+use function Sentry\logger;
+
+logger()->info('Adding item %s for user %s', [$item_id, $user_id], ['extra' => 123]);
+
+logger()->emit(
+  level: LogLevel::info(),
+  message: 'Adding item for user',
+  attributes: [
+    'item_id': item_id,
+    'user_id': user_id,
+    'extra': 123
+  ],
+);
+```
+
+#### Java
+
+```java
+import io.sentry.Sentry;
+
+// example with MessageFormat based string template
+Sentry.logger().info("Adding item {0,string} for user {1,string}", item_id, user_id);
+
+Sentry.logger().emit(log -> {
+  log.setLevel("info");
+  log.setMessage("Adding item for user");
+  log.setAttribute("item_id", item_id);
+  log.setAttribute("user_id", user_id);
+  log.setAttribute("extra", 123);
+});
+
+// Java - Builder Pattern, where setters return builder for method chaining
+Sentry.Logger.emit(log -> {
+  log.setLevel("info")
+  .setMessage("Adding item for user")
+  .setAttribute("item_id", item_id)
+  .setAttribute("user_id", user_id)
+  .setAttribute("extra", 123)
+});
+
+// Kotlin
+Sentry.Logger.emit("Adding item for user") { // fun emit(block: LogItem.() -> Unit)
+  setLevel("info")
+  setAttribute("item_id", item_id)
+  setAttribute("user_id", user_id)
+  setAttribute("extra", 123)
+}
+```
+
+#### Apple
+
+```swift
+// Swift
+SentrySDK.logger
+  .info(message: "Adding item %s for user %s",
+    params: [itemId, userId],
+    attributes: ["extra": @"123"]
+  )
+
+// Objective-C
+[SentrySDK.logger
+    emit :@"Adding item for user"
+    level: @"info"
+    attributes: @{ @"extra" : @"123", @"item_id" : item_id, @"user_id" : user_id }
+];
```

Comment:
We can ditch ObjC. I would like to write the API in Swift and not worry too much about how the API looks in ObjC.
```suggestion
```

Reply:
Done with 6dc07e630fefce60d3dc66d74be0ae761083af31